plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreProcedure.java

```diff
@@ -311,6 +319,22 @@ public void setKind(PostgreProcedureKind kind) {
     public long getObjectId() {
         return oid;
     }
+    
+    public long getXmin() {
+        return xmin;
+    }
```

Comment:
I think we can use @Property annotation to show it in procedure metadata.

Reply:
Can you show me where I can see thap procedure metadata?
I added annotation you said

Reply:
When you open some function, it's metadata is in the top panel of properties view. Probably xmin will be somewhere here
![image](https://user-images.githubusercontent.com/28875055/214130948-7ee3d63b-9f07-4406-8243-513e72ce266d.png)

Reply:
thank you !